ee_tests/src/support/core.ts

```diff
@@ -195,12 +195,13 @@ export class SpaceName {
   static spaceName: string;
 
   static newSpaceName(): string {
-    let d = new Date();
-    let month = d.toLocaleString('en-US', { month: 'short' }).toLowerCase();
-    let day = d.getDate();
-    let time = d.getTime();
-    let spaceName = `test${month}${day}${time}`;
-
+    const d = new Date();
+    const month = ( d.getUTCMonth() < 9 ) ? `0${d.getUTCMonth() + 1}` : `${d.getUTCMonth() + 1}`;
```

Comment:
What about `(d.getUTCMonth()+1).toString().padStart(2, '0')`? This would require us to upgrade Node.js version in `Dockerfile.builder` (e.g. to version used in other OSIO projects - 8.3.0)

Reply:
BTW upgrade of Node.js is in backlog, see #479

Reply:
One step at a time :)